docs/Project-Lifecycle.md

```diff
@@ -324,6 +325,20 @@ Then you are able to review the tests results in [Test results page](http://comm
 
 To test with Paypal, you need own Sandbox Buyer credentials via a developer account.
 
+### Klarna Testing notes
+
+Klarna has very strict requirements to the payment details. For testing purpose one should use
+[special Klarna test credentials](https://developers.klarna.com/en/de/kpm/test-credentials).
+
+These tests require next CTP settings:
+
+  * `gender`, `ip` and `birthday` custom fields in _payment-INVOICE-KLARNA_ custom type
+  * German language, country and prices must be active.
+  * 19% German tax is active
```

Comment:
You should setup manually only currency, language and country as prices are part of the product and can be created over API (IT -Tests). Also tax category can be created and assigned over API (test)